src/basket/svgs/empty-basket.svg

```diff
@@ -0,0 +1,45 @@
+<svg width="295" height="190" viewBox="-25 0 320 190" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Shouldn't this be in franklin?

Reply:
no-blast-result was in uniprot website so I put that one here too

Reply:
We also have all the illustrations in `src/images`. I'd assume icons in franklin, big specific illustrations in uniprot-website